server/fixtures/mockUser.js

```diff
@@ -0,0 +1,15 @@
+import database from "../models/index";
+
+const User = database["Users"];
```

Comment:
This is not a fixture, This is meant to be just the new User object. Please sync with @Ndifreke